polipy/polipy.py

```diff
@@ -39,6 +40,7 @@ def __init__(self, url):
         self.url['url'] = url
         self.url = self.url | parse_url(url)
         self.url['domain'] = self.url['domain'].strip().strip('.').strip('/')
+        self.html_file = html_file
```

Comment:
Yeah, I think this is fine for now. Maybe ultimately we could merge the `self.htm_file` better into the `Polipy` object, as that's basically the exact same attribute as the `self.source['dynamic_html']` after the scraping part. But otherwise it's good to have an option of either providing an URL (and then scraping) or the already-scraped HTML to the constructor.